### PR TITLE
draft for mo paddle docs

### PR DESCRIPTION
**Pull request by meiyang-intel**
paddle mo docs for internal review

### CODE REVIEW

**Comment by meiyang-intel** [comment]
Do we need to continually maintain the docs for mo_paddle in the future?

**Comment by meiyang-intel** [comment]
Do we need to add docs for each models like docs/MO_DG/prepare_model/convert_model/tf_specific/*.md ?

**Comment by yuxu42** [comment]
> Do we need to continually maintain the docs for mo_paddle in the future?

yes. and more models are planned to be supported.

File: docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md [hunk 1]
```diff
@@ -519,3 +519,7 @@ Standard ONNX\* operators:
 | Upsample | No |
 | Where | No |
 | Xor | No |
+
+## Paddle\* Supported Layers
```
Comment (ceciliapeng2011) [inline docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md]: This has included by LiuBo's PR: https://github.com/openvinotoolkit/openvino/pull/6687. So maybe we can leave it here?

File: docs/MO_DG/prepare_model/Config_Model_Optimizer.md [hunk 2]
```diff
@@ -151,6 +155,10 @@ pip3 install -r requirements_kaldi.txt
     *   To install dependencies only for ONNX:
 ```shell
 pip3 install -r requirements_onnx.txt
+```
+    *   To install dependencies only for Paddle:
+```shell
+pip3 install -r requirements_paddle.txt
```
Comment (ceciliapeng2011) [inline docs/MO_DG/prepare_model/Config_Model_Optimizer.md]: Where is requirements_paddle.txt?

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+> **NOTE:** It is necessary to specify mean and scale values for most of the Caffe\* models to convert them with the Model Optimizer. The exact values should be determined separately for each model. For example, for Caffe\* models trained on ImageNet, the mean values usually are `123.68`, `116.779`, `103.939` for blue, green and red channels respectively. The scale value is usually `127.5`. Refer to [Framework-agnostic parameters](Converting_Model_General.md) for the information on how to specify mean and scale values.
+
+## Convert a Paddle* Model <a name="Convert_From_Paddle"></a>
+
+To convert a Paddle\* model:
+
+1. Go to the `$INTEL_OPENVINO_DIR/deployment_tools/model_optimizer` directory.
+2. Use the `mo.py` script to simply convert a model, specifying the path to the input model `.pdmodel` file and the path to an output directory with write permissions:
+```sh
+python3 mo.py --input_model <INPUT_MODEL>.pdmodel --output_dir <OUTPUT_MODEL_DIR>
```
Comment (ceciliapeng2011) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md]: We need specify "--framework paddle"

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# Converting a Paddle* Model {#openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_Paddle}
+
+A summary of the steps for optimizing and deploying a model that was trained with Paddle\*:
+
+1. [Configure the Model Optimizer](../Config_Model_Optimizer.md) for Paddle\*.
+2. [Convert a Paddle\* Model](#Convert_From_Paddle) to produce an optimized [Intermediate Representation (IR)](../../IR_and_opsets.md) of the model based on the trained network topology, weights, and biases values
+3. Test the model in the Intermediate Representation format using the [Inference Engine](../../../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) in the target environment via provided Inference Engine [sample applications](../../../IE_DG/Samples_Overview.md) // sample?
+4. [Integrate](../../../IE_DG/Samples_Overview.md) the [Inference Engine](../../../IE_DG/Deep_Learning_Inference_Engine_DevGuide.md) in your application to deploy the model in the target environment
+
+## Supported Topologies
```
Comment (ceciliapeng2011) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md:10]: Please update to top10 NN requested by Baidu.

**Pull request closed without merging**